physionet-django/project/views.py

```diff
@@ -2307,3 +2312,25 @@ def generate_signed_url(request, project_slug):
     GCPLog.objects.update_or_create(project=project, user=request.user, data=data)
 
     return JsonResponse({'url': url})
+    
+class PublishedProjectList(mixins.ListModelMixin, generics.GenericAPIView):
+    """
+    List all Active Projects
+    """
+    queryset = PublishedProject.objects.all().order_by('id')
+    permission_classes = [IsAuthenticated]
```

Comment:
I'm not familiar with "IsAuthenticated". Might be good to have a quick chat about this! I'd like to understand how "IsAuthenticated" relates to the Django permissions that we're using.

Reply:
Sure, I will also edit the pull request to add some details to it.